<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesWidget</class>
 <widget class="QDockWidget" name="NotesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="2">
     <widget class="QTextEdit" name="noteEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QFrame" name="buttonFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="editButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Edit Note</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="titleEdit"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Note Title:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="fetchButton">
      <property name="text">
       <string>Fetch Notes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>m/z</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>rt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>noteEdit</sender>
   <signal>textChanged()</signal>
   <receiver>buttonFrame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>titleEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buttonFrame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
